/tmp/tmp2op4zbsd:227:82-83: error: syntax error, unexpected (

*** ERROR: (gringo): grounding stopped because of errors
Traceback (most recent call last):
  File "test.py", line 167, in <module>
    actions = grounding.ground_actions()
  File "/home/xoxo/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 39, in ground_actions
    model = self._solve_lp()
  File "/home/xoxo/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 50, in _solve_lp
    model_filename = run_clingo(lp)
  File "/home/xoxo/Applications/tarski/src/tarski/reachability/clingo_wrapper.py", line 26, in run_clingo
    raise ExternalCommandError("Gringo exited with code {}".format(retcode))
tarski.errors.ExternalCommandError: Gringo exited with code 1
